src/api.ts

```diff
@@ -209,12 +209,20 @@ export default class Chromeless<T extends any> implements Promise<T> {
     return this
   }
 
-  cookiesClear(name: string): Chromeless<T> {
-    throw new Error('Not implemented yet')
+  cookiesDelete(name: string, url: string): Chromeless<T> {
+    if (typeof name === 'undefined') {
+      throw new Error('Cookie name should be defined.')
+    }
```

Comment:
We could probably have this method lookup the URL since it'll likely be ran in the context of the page it's in. This is good for now, but might be a nice change/improvement later on!